test/devfiles/v200/commands/ExecBadDefault.yaml

```diff
@@ -0,0 +1,18 @@
+  - id: testExec1
+    exec:
+      attributes: 
+        test: Exec Test test
+        scope: Api
+      commandLine: 'echo "Hello ${GREETING} ${USER}"'
+      component: api-tests
+      group: 
+        isDefault: maybe
+        kind: test
+      hotReloadCapable: false
+      label: "Command Exec test"
+      env:
+        - name: "USER"
+          value: "Test Tester"
+        - name : "GREETING"
+          value : "Hello" 
+      workingDir: $PROJECTS_ROOT  
```

Comment:
Could you please add newlines in files?

Reply:
Sorry, not sure what you mean.

Reply:
I mean this 
![Screenshot_20201111_171812](https://user-images.githubusercontent.com/5887312/98829496-f06bc180-2441-11eb-8cd8-052719d1d5cd.png)

here you can find why it's better to add it https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725

Reply:
Thank you  all done:
- added newlines
- updated tests for recent changes - attributes updates and name and id restrictions
- added tests for. the new name and id restrictions 

Read for review again, hopefully all good this time :-)

Reply:
Some tests failing on the build. failing to find files - need to dig into it, all was good locally on my mac.

Reply:
All good now, difference in how Mac and linux handle case in file names.